contracts/treasury/LivepeerGovernor.sol

```diff
@@ -0,0 +1,185 @@
+ * GovernorCountingOverridable extension.
+ */
+contract LivepeerGovernor is
+    ManagerProxyTarget,
+    Initializable,
+    GovernorUpgradeable,
+    GovernorSettingsUpgradeable,
+    GovernorTimelockControlUpgradeable,
+    GovernorVotesUpgradeable,
+    GovernorVotesQuorumFractionUpgradeable,
+    GovernorCountingOverridable
+{
+    /**
+     * @notice TreasuryGovernor constructor. Only invokes constructor of base Manager contract with provided Controller.
+     * @dev This constructor will not initialize any state variables besides `controller`. The `initialize` function
+     * must be called through the proxy after construction to initialize the contract's state in the proxy contract.
+     * @param _controller Address of Controller that this contract will be registered with
+     */
+    /// @custom:oz-upgrades-unsafe-allow constructor
+    constructor(address _controller) Manager(_controller) {
+        _disableInitializers();
+    }
+
+    /**
+     * Initializes the LivepeerGovernor instance. This requires the following contracts to have already been deployed
+     * and registered on the controller:
+     * - "Treasury" (TimelockControllerUpgradeable)
+     * - "BondingCheckpointsVotes"
+     * - "PollCreator"
+     */
+    function initialize(
+        uint256 initialVotingDelay,
+        uint256 initialVotingPeriod,
+        uint256 initialProposalThreshold
+    ) public initializer {
+        __Governor_init("LivepeerGovernor");
+        __GovernorSettings_init(initialVotingDelay, initialVotingPeriod, initialProposalThreshold);
+        __GovernorTimelockControl_init(treasury());
+
+        // The GovernorVotes module will hold a fixed reference to the votes contract. If we ever change its address we
+        // need to call the {bumpVotesAddress} function to update it in here as well.
+        __GovernorVotes_init(votes());
+
+        // Initialize with the same value from the existing polling system.
+        uint256 initialQuorum = pollCreator().QUORUM();
+        __GovernorVotesQuorumFraction_init(initialQuorum);
+
+        __GovernorCountingOverridable_init();
+    }
+
+    /**
+     * @dev Overrides the quorum denominator from the GovernorVotesQuorumFractionUpgradeable extension. We use
+     * MathUtils.PERC_DIVISOR so that our quorum numerator is a valid MathUtils fraction.
+     */
+    function quorumDenominator() public view virtual override returns (uint256) {
+        return MathUtils.PERC_DIVISOR;
+    }
+
+    /**
+     * @dev See {GovernorCountingOverridable-votes}.
+     */
+    function votes() public view override returns (IVotes) {
+        return bondingCheckpointVotes();
+    }
+
+    /**
+     * @dev See {GovernorCountingOverridable-quota}. We use the same QUOTA value from the protocol governance system for
+     * now, but can consider changing this in the future (e.g. to make it updateable through proposals without deploys).
+     */
+    function quota() public view override returns (uint256) {
```

Comment:
See previous comment re: independent state variables for LivepeerGovernor for quorum and quota.